<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysTrayWidget</class>
 <widget class="QWidget" name="SysTrayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QUsb2Snes</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2,1,2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/deviceicon.svg&quot; width=&quot;20&quot; height=&quot;20&quot; style=&quot;vertical-align: middle&quot;/&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Device(s)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="devicesLabel">
       <property name="text">
        <string>Devices status</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/appicon.svg&quot; width=&quot;20&quot; height=&quot;20&quot; style=&quot;vertical-align: middle&quot;/&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Application(s)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="applicationLabel">
       <property name="text">
        <string>Applications connected</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ressources.qrc">:/img/icon64x64.ico</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Menu</string>
       </property>
       <property name="icon">
        <iconset resource="../ressources.qrc">
         <normaloff>:/img/settings.svg</normaloff>:/img/settings.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
